### ISSUE TITLE
Layout gotten larger with newer versions

### ISSUE THREAD

**Issue by fribse** [custom-cards/bar-card#74]
Hey there

I love the bar card, and I've built an overview of the battery states of devices.
But the later versions has made the layout a bit weird, and I can't figure out what's going on.
I must have missed a change somewhere.

It now looks like this:
![image](https://user-images.githubusercontent.com/13660621/80845465-9031d280-8c09-11ea-8f37-33e990c4f91a.png)
The space between the bars is not supposed to be there :-)

The code looks like this:
```
card:
  align: left
  columns: 2
  padding: 2px
  positions:
    icon: 'off'
    name: inside
  severity:
    - color: '#bf4040'
      value: 10
    - color: '#bf9540'
      value: 30
    - color: '#40bf40'
      value: 100
  title_position: inside
  title_style:
    font-size: 16px
  type: 'custom:bar-card'
  value_style:
    font-size: 8px
filter:
  exclude:
    - entity_id: '*mobil*'
    - entity_id: '*h8*'
  include:
    - entity_id: '*_battery'
    - entity_id: '*_battery_level'
type: 'custom:auto-entities'
```

Any input on what I've missed?

**Comment by Gluwc** [comment]
Are you using this inside a horizontal stack or panel view?

**Comment by fribse** [comment]
The 'full layout' looks like this:
![image](https://user-images.githubusercontent.com/13660621/80858053-06105b00-8c57-11ea-860d-ce06804c5dd2.png)

It's in a panel view with three columns in a horizontal stack, so both :-)

**Comment by Gluwc** [comment]
Should be fixed in [3.1.6](https://github.com/custom-cards/bar-card/releases/tag/3.1.6).

**Comment by fribse** [comment]
WOW, that was fast! And yes, it's solved!
